### DISCUSSION TITLE
What does HCL mean?

### DISCUSSION THREAD

**Discussion by nvictor** [ariga/atlas#1830] [Q&A] [answered]
I am quite confused and perplexed. The term HCL is all over the documentation and not defined anywhere. Is it related to the HashiCorp Configuration Language?

Thanks

**Comment by a8m** [comment, marked as answer]
Hey @nvictor 👋 
Yes, Atlas has a flavor of Hashicorp configuration language and you can check it here: https://atlasgo.io/atlas-schema/hcl

Please let me know if there are specific areas on the website where we could provide more clarity.
